javalib/src/main/scala/java/lang/process/UnixProcess.scala

```diff
@@ -6,7 +6,9 @@ private[lang] abstract class UnixProcess extends GenericProcess {}
 
 object UnixProcess {
   def apply(builder: ProcessBuilder): Process = {
-    val useGen2 = if (LinktimeInfo.is32BitPlatform) {
+    val useGen2 = if (!LinktimeInfo.isCPPRuntimeEnabled) {
+      true
+    } else if (LinktimeInfo.is32BitPlatform) {
```

Comment:
Now, it would be nice to be able to link-time fail if you don't use the cpp runtime on a 32 bit machine.
Is it possible? We could ask the user to enable the cpp runtime if they want to use `java.lang.process.Process` on that platform.

Reply:
I looked at the process code and it was just the collections (Map) that was C++. It seems that could be replaced to make that pure C too.

Reply:
Then I have to check if I can migrate that to c as well.

Reply:
Converted process_monitor to C, using simple linked lists

Reply:
I had poked around and found this which has a liberal license - https://github.com/troydhanson/uthash